pkg/common/utils/resources.go

```diff
@@ -73,14 +73,14 @@ var (
 
 	// These defaults allow up to a PostgreSQL or Redis standard-8 with one replica.
 
-	// defaultCPURequests 2* standard-8 will request 4 CPUs. This default has 500m as spare for jobs
-	DefaultCPURequests = resource.NewMilliQuantity(4500, resource.DecimalSI)
-	// defaultCPULimit by default same as DefaultCPURequests
-	DefaultCPULimit = DefaultCPURequests
-	// defaultMemoryRequests 2* standard-8 will request 16Gb. This default has 500mb as spare for jobs
-	DefaultMemoryRequests = resource.NewQuantity(17301504000, resource.BinarySI)
-	// defaultMemoryLimits same as DefaultMemoryRequests
-	DefaultMemoryLimits = DefaultMemoryRequests
+	// defaultCPURequests 2* standard-8 will request 4 CPUs + the sidecars will each request 850m. This default has 500m as spare for jobs
```

Comment:
I would not hardcode the sidecar limits, simply because not all of our services have sidecars.

For example this would give redis much more limits than it needs because it doesn't have sidecars.

I probably would have added it to the comp-function `AddInitialNamespaceQuotas` and inject the overhead depending on the service.

Reply:
Did a lot of refactoring and changes here. Please check again